pkgs/development/compilers/llvm/common/bolt/setup-hook.sh.in

```diff
@@ -0,0 +1,16 @@
+# shellcheck shell=bash
+
+function boltFixupPhase {
+  local i
+  while IFS= read -r -d $'\0' i; do
+    if ! isELF "$i"; then continue; fi
+    if ! @readelf@ --program-headers "$i" | grep -q LOAD; then continue; fi
```

Comment:
Is this robust enough? "LOAD" is enough of a pattern to not have false positives?

Reply:
I think so? LOAD indicates shared libraries I believe so that's what I used to determine if there's any.

Reply:
And the string "LOAD" will never appear anywhere else in this output?

Reply:
Afaik, it shouldn't